apis_ontology/management/commands/importproperties.py

```diff
@@ -0,0 +1,32 @@
+# code taken from https://github.com/acdh-oeaw/apis-instance-sicprod/blob/cd203192fc0cb6d44ebbfba37c5a572f6b9a1bd3/apis_ontology/management/commands/importproperties.py
+import json
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+from apis_core.apis_relations.models import Property
+from django.contrib.contenttypes.models import ContentType
+
+
+class Command(BaseCommand):
+    help = "Import properties from data/properties.json"
+
+    def handle(self, *args, **options):
+        base_path = Path(__file__).resolve().parent.parent.parent.parent
+        properties_json = base_path / "data/properties.json"
+        data = json.loads(properties_json.read_text())
+
+        for p in data:
+            prop, created = Property.objects.get_or_create(pk=p["id"])
+            prop.name = p["name"]
+            prop.name_reverse = p["name_reverse"]
+            prop.save()
+            prop.obj_class.clear()
+            for obj in p["obj"]:
+                prop.obj_class.add(ContentType.objects.get(model=obj.lower()))
+            prop.subj_class.clear()
+            for subj in p["subj"]:
+                prop.subj_class.add(ContentType.objects.get(model=subj.lower()))
+            if created:
+                print(f"Created {prop}")
+            else:
+                print(f"Updated {prop}")
```

Comment:
Could you add a trailing newline at the end of the file and rename it to `import_properties.py`?

I know [PEP8](https://peps.python.org/pep-0008/#package-and-module-names) leaves room for interpretation regarding "readability" of file names, but find that using underscores as word separators helps (me) a lot with parsing them.

Reply:
updated it